### ISSUE TITLE
`Cart Line Items » showRemoveItemLink`: Add code snippet and screenshot

### ISSUE THREAD

**Issue by nielslange** [woocommerce/woocommerce-gutenberg-products-block#10901]
Recently, we've updated the "Available Extensibility Interfaces for the Cart and Checkout blocks" post. In this post, we introduced new code snippets and screenshots. This issue aims to add the following code snippet and screenshot to the [`Cart Line Items » showRemoveItemLink`](https://github.com/woocommerce/woocommerce-blocks/blob/6083484477410fd99d1efd5ca909288eb4d19963/docs/third-party-developers/extensibility/checkout-block/available-filters.md?plain=1#L26) section:

```js
const { registerCheckoutFilters } = window.wc.blocksCheckout;
 
// Show remove item link of the cart line items.
registerCheckoutFilters( 'example-extension', {
  showRemoveItemLink: ( value, extensions, args ) => {
    // Return early since this filter is not being applied in the Cart context.
    // We must return the original value we received here.
    if ( args?.context !== 'cart' ) {
      return value;
    }
    return false;
  }
} );
```

<img width="1512" alt="screenshot-2023-08-24-at-13 19 35" src="https://github.com/woocommerce/woocommerce-blocks/assets/3323310/763fabd2-96f1-4854-905b-c1f9f8fa2870">

---

> **Note**
> The code snippet and screenshot should be placed below the table with the filter names and the sentence _"Each of these filters has the following arguments [...]"_, and should have the following subheading
```md
### showRemoveItemLink
```

**Comment by corners2wall** [comment]
@nielslange  Assign to me pls

**Comment by nielslange** [comment]
> @nielslange Assign to me pls

Thanks for volunteering to work on this task. As I've just assigned #10898 to you, let's focus on that one first, before working on this issue. See also https://github.com/woocommerce/woocommerce-blocks/issues/10898#issuecomment-1716889695.

**Comment by masteradhoc** [comment]
@nielslange you can assign this one to me. it will be solved within #10969 

Before Screenshot
![image](https://github.com/woocommerce/woocommerce-blocks/assets/6242098/ec76f525-e2cf-468c-9634-ae9e84e5aa48)

After Screenshot
![image](https://github.com/woocommerce/woocommerce-blocks/assets/6242098/3171cd92-33f1-4182-b875-bc7e422771a0)
